helpers/forbidden_gene_check.py

```diff
@@ -58,9 +58,10 @@ def read_json_from_path(bucket_path: str | Path | None, default: Any = None) ->
     if isinstance(bucket_path, str):
         bucket_path = to_path(bucket_path)
 
-    if bucket_path.exists():
-        with bucket_path.open() as handle:
-            return json.load(handle)
+    if isinstance(bucket_path, Path):
```

Comment:
Do you want an else fail here if you don't pass a string bucket path?

Reply:
Yup, from the type hinting all supported types are covered, but I've chucked a ValueError if the arg is an invalid type 👍